frontend/app/controllers/admin/badges/list.js

```diff
@@ -30,6 +32,12 @@ export default Controller.extend({
           if (records.length > 0) {
             this.set('page', filter.page);
             this.set('badges', records);
+            if (records.length < 10) {
```

Comment:
These actions will gets execute when clicked on the actions and not on coming to the page. Please override the `setupController` in the route also to implement this change

Reply:
done @yashLadha